systems/data-plan/package/pkg/server/package.go

```diff
@@ -186,6 +189,10 @@ func (p *PackageServer) Add(ctx context.Context, req *pb.AddPackageRequest) (*pb
 		PackageDetails: db.PackageDetails{
 			Apn: req.Apn,
 		},
+		Overdraft:     req.Overdraft,
+		TrafficPolicy: req.TrafficPolicy,
+		Networks:      req.Networks,
+		SyncStatus:    types.SyncStatusPending,
```

Comment:
What is this for ?

Reply:
We use that to update the resource status because the second part of the request is asynchronous (event based). We that in order to provide the right  resource status, whenever the caller request it.